tests/backup/backup_helper.go

```diff
@@ -1960,3 +1960,34 @@ func CreateNamespaceLabelScheduleBackupWithoutCheck(scheduleName string, cluster
 	}
 	return resp, nil
 }
+
+// AddLabelsToMultipleNamespaces add labels to multiple namespace
+func AddLabelsToMultipleNamespaces(labels map[string]string, namespaces []string) error {
+	for _, namespace := range namespaces {
+		err := Inst().S.AddNamespaceLabel(namespace, labels)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// GenerateRandomLabels creates random label
+func GenerateRandomLabels(number int) map[string]string {
+	labels := make(map[string]string)
+	for i := 0; i < number; i++ {
+		key := fmt.Sprintf("%v-%v", i, uuid.New())
+		value := uuid.New()
+		labels[key] = value
+	}
+	return labels
+}
+
+// mapToString will create a string from map
+func mapToString(m map[string]string) string {
+	var pairs []string
+	for k, v := range m {
+		pairs = append(pairs, k+"="+v)
+	}
+	return strings.Join(pairs, ",")
+}
```

Comment:
`mapToKeyValueString` might be a more intuitive name, and the comment as `// mapToKeyValueString converts a map of  string keys and string value to a comma separated string of "key=value"`.

Reply:
Done